bigtable/google/cloud/bigtable/row_set.py

```diff
@@ -61,11 +61,13 @@ def _update_message_request(self, message):
 class RowRange(object):
     """ Convenience wrapper of google.bigtable.v2.RowRange
 
-    :type start_key: string utf-8
-    :param start_key: start key of the row range
+    :type start_key: bytes
+    :param start_key: (Optional) Start key of the row range. If left empty,
+                      will be interpreted as the empty string.
 
-    :type end_key: string utf-8
-    :param end_key: end key of the row range
+    :type end_key: bytes
+    :param end_key: (Optional) End key of the row range. If left empty,
+                    will be interpreted as the infinite string.
```

Comment:
I don't understand what "infinite string" means here.  I think the semantics are that there is *no* end to the row range.

Reply:
Updated the doc string.